src/plugins/orangehrmClaimPlugin/entity/ClaimExpense.php

```diff
@@ -0,0 +1,192 @@
+     * @var ExpenseType
+     * @ORM\ManyToOne(targetEntity="OrangeHRM\Entity\ExpenseType")
+     * @ORM\JoinColumn(name="expense_type_id", referencedColumnName="id")
+     */
+    private ExpenseType $expenseType;
+
+    /**
+     * @var DateTime
+     * @ORM\Column(name="date", type="datetime")
+     */
+    private DateTime $date;
+
+    /**
+     * @var float
+     * @ORM\Column(name="amount", type="float", nullable=false)
+     */
+    private float $amount;
+
+    /**
+     * @var string|null
+     * @ORM\Column(name="note", type="string", nullable=true, length=1000)
+     */
+    private ?string $note;
+
+    /**
+     * @var ClaimRequest
+     * @ORM\ManyToOne(targetEntity="OrangeHRM\Entity\ClaimRequest", inversedBy="claimExpenses")
+     * @ORM\JoinColumn(name="request_id", referencedColumnName="id")
+     */
+    private ClaimRequest $claimRequest;
+
+    /**
+     * @var bool
+     * @ORM\Column(name="is_deleted", type="boolean", nullable=false)
+     */
+    private bool $isDeleted = false;
+
+    /**
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @param int $id
+     */
+    public function setId(int $id): void
+    {
+        $this->id = $id;
+    }
+
+    /**
+     * @return ExpenseType
+     */
+    public function getExpenseType(): ExpenseType
+    {
+        return $this->expenseType;
+    }
+
+    /**
+     * @param ExpenseType $expenseType
+     */
+    public function setExpenseType(ExpenseType $expenseType): void
+    {
+        $this->expenseType = $expenseType;
+    }
+
+    /**
+     * @return DateTime
+     */
+    public function getDate(): DateTime
+    {
+        return $this->date;
+    }
+
+    /**
+     * @param DateTime $date
+     */
+    public function setDate(DateTime $date): void
+    {
+        $this->date = $date;
+    }
+
+    /**
+     * @return float
+     */
+    public function getAmount(): float
+    {
+        return $this->amount;
+    }
+
+    /**
+     * @param float $amount
+     */
+    public function setAmount(float $amount): void
+    {
+        $this->amount = $amount;
+    }
+
+    /**
+     * @return ?string
+     */
+    public function getNote(): ?string
+    {
+        return $this->note;
+    }
+
+    /**
+     * @param ?string $note
+     */
+    public function setNote(?string $note): void
+    {
+        $this->note = $note;
+    }
+
+    /**
+     * @return ClaimRequest
+     */
+    public function getClaimRequest(): ClaimRequest
+    {
+        return $this->claimRequest;
+    }
+
+    /**
+     * @param ClaimRequest $claimRequest
+     */
+    public function setClaimRequest(ClaimRequest $claimRequest): void
+    {
+        $this->claimRequest = $claimRequest;
+    }
+
+    /**
+     * @return bool
+     */
+    public function isDeleted(): bool
+    {
+        return $this->isDeleted;
+    }
+
+    /**
+     * @param bool $isDeleted
+     */
+    public function setIsDeleted(bool $isDeleted): void
```

Comment:
```suggestion
    public function setDeleted(bool $deleted): void
```

Reply:
change in the `html/ohrm_claim_expense/orangehrm-5.4/src/plugins/orangehrmClaimPlugin/entity/ClaimRequest.php` as well